doc/source/CHANGELOG.rst

```diff
@@ -7,6 +7,15 @@ The format is largely inspired by keepachangelog_.
 
 .. _0.1.1:
 
+v0.6.5 - Unreleased
```

Comment:
I think this is a new feature so it should be added to version 0.7.0. If we do that, we are going to update libjampp

Reply:
Sure.